internal/providers/terraform/aws/testdata/kinesis_firehose_delivery_stream_test/kinesis_firehose_delivery_stream_test.golden

```diff
@@ -0,0 +1,48 @@
+
+ Name                                                         Monthly Qty  Unit                    Monthly Cost 
```

Comment:
Were you able to double check these prices vs the calculator?

Reply:
@tim775 Yep!